build/android.gni

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file contains feature flags for Crosswalk Android build. Users should
+# import it into their build folder by run:
+#   echo 'import("//xwalk/build/android.gni")' > out/your_dir/args.gn'
+# Then run:
+#   gn gen out/your_dir
+#   ninja -C out/your_dir xwalk_builder
+# to generate Android Apks of Crosswalk.
+
+import("//xwalk/build/common.gni")
+
+target_os = "android"
```

Comment:
I'd rather not do this here and have users explicitly do this in `args.gn`. It makes it easier to see what's going on with `gn args`, is more in line with existing Chromium GN instructions for Android and also allows us to get rid of `android.gni` in the future more easily.